sdk/helper/salt/salt.go

```diff
@@ -115,6 +115,23 @@ func NewSalt(ctx context.Context, view logical.Storage, config *Config) (*Salt,
 	return s, nil
 }
 
+// NewNonpersistentSalt creates a new salt with default configuration and no storage usage.
+func NewNonpersistentSalt() *Salt {
+	// Setup the configuration
+	config := &Config{}
+	config.Location = ""
+	config.HashFunc = SHA256Hash
+	config.HMAC = sha256.New
+	config.HMACType = "hmac-sha256"
+
+	s := &Salt{
+		config: config,
+	}
+	s.salt, _ = uuid.GenerateUUID()
```

Comment:
is there a scenario where we care about the `GenerateUUID()` err?

Reply:
I thought about that but we really don't care what the salt is.  The worst case is we end up with "" and so the hash is predictable, but we don't actually hash anything in the test message? For now, anyway.